supplier/tests/test_supplier_page.py

```diff
@@ -13,6 +36,29 @@ def non_supplier_mock():
     return mock
 
 
+@pytest.fixture
+def request_searched_product_mock():
+    mock = Mock()
+    mock.GET = {}
+    return mock
+
+
+@pytest.fixture
+def render_mock(monkeypatch):
+    mock = Mock()
+    monkeypatch.setattr(views, views.render.__name__, mock)
+    return mock
```

Comment:
Not sure if thwre is a need to return anything here

Reply:
@Omeramsc In case you want to assert on the mock calls.

Reply:
right, good point. resolving